web/shapeworks/package.json

```diff
@@ -13,8 +13,9 @@
     "@vue/composition-api": "^1.4.6",
     "core-js": "^3.6.5",
     "django-s3-file-field": "^0.1.1",
+    "itk": "^14.1.1",
     "shader-loader": "^1.3.1",
-    "vtk.js": "^17.2.6",
+    "vtk.js": "^22.0.0",
```

Comment:
Would it be worth just using "*" so we don't have to continually update this file?  vtk.js updates often and the lock file will at last show the last version that we knowingly worked with.  It has had a very stable interface.

Reply:
Sure, we can do that. Changed in [cbd683d](https://github.com/girder/shapeworks-cloud/pull/134/commits/cbd683d42aa18be1e6179e79a04edfd24052bebf)